src/Rector/AbstractRector.php

```diff
@@ -200,6 +200,8 @@ final public function enterNode(Node $node)
 
         $this->printDebugCurrentFileAndRule();
 
+        $this->changedNodeScopeRefresher->reIndexNodeAttributes($node);
+
```

Comment:
This used to decrease performance quiet a bit.
That's why it should run only when node changed. 

How does it affect speed now?

Reply:
This is not refresh Scope, only reindex stmts early, params, closure uses. array_values() should not take many performance degradation, except per node has 1000+ subnodes.

Reply:
Ok, let's ship it :+1: